redis/src/main/scala/zio/redis/Interpreter.scala

```diff
@@ -75,6 +77,27 @@ trait Interpreter {
           .tapError(e => logger.error(s"Executor exiting: $e"))
     }
 
+    private final class InMemory(
```

Comment:
I'd remove `()`.

Reply:
Makes sense for now. I was thinking that it will have STM data structures in it

Reply:
Ah yes, true. Well, you'll introduce it anyhow, so this is a no-op.